src/cmdstan-guide/pathfinder_config.Rmd

```diff
@@ -53,6 +55,17 @@ Can differ from `num_psis_draws`.
 - `num_elbo_draws` - Number of Monte Carlo draws to evaluate ELBO.
 Must be a positive integer.  Default value is $25$.
 
+- `psis_resample` - If True ($1$), perform psis resampling on samples returned
```

Comment:
This is an unfortunate choice of names.  "psis" is Pareto-smoothed importance sampling, but we're not doing importance sampling, we're doing importance resampling.  So it should be `psi_resample`.  In the literature, it's called "sampling importance resampling" or SIR.  I'd use "PSIS" as the acronym in text.